toolkit/tools/imagecustomizerapi/bootloadertype.go

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package imagecustomizerapi
+
+import (
+	"fmt"
+)
+
+type BootLoaderType string
+
+const (
+	BootLoaderTypeDefault     BootLoaderType = ""
```

Comment:
Should the default be set to one of the other values? (grub2 or systemd-boot)

Reply:
Or is it intended to be like the reset type? i.e. the default actually means 'undefined' or 'not-set'?
If so, then, we can leave it empty, but rename the constant name to something like `BootLoaderTypeUndefined`